dnf5/commands/module/module_list.cpp

```diff
@@ -64,8 +74,20 @@ void ModuleListCommand::run() {
         query.filter_disabled();
     }
 
-    output::print_modulelist_table(query.list());
-    output::print_modulelist_table_hint();
+    print(query);
+    if (!query.empty()) {
+        output::print_modulelist_table_hint();
+    }
+
+    if (!unmatched_module_spec.empty()) {
+        for (auto const & module_spec : unmatched_module_spec) {
+            std::cerr << libdnf5::utils::sformat(_("No matches found for \"{}\"."), module_spec) << std::endl;
+        }
+    }
```

Comment:
This reminds me of our discussion on the consistency of exit codes across various `info` and `list` commands. I've tasked myself with creating a follow-up on this.

Reply:
Nice, I haven't started yet, I'll pass it over to you then. Thanks!

Reply:
Issue: https://github.com/rpm-software-management/dnf5/issues/1075